TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleNew.java

```diff
@@ -38,134 +38,87 @@ public void init() {
 
     @Override
     public void loop() {
+            //Neutral condition for vertical
+        if (gamepad1.left_stick_y > -0.3 && gamepad1.left_stick_y < 0.3) { wheelMove (0, 0, 0, 0); }
 
-        if (gamepad1.left_stick_y > -0.3 && gamepad1.left_stick_y < 0.3) {
-            wheel1.setPower(0);
-            wheel2.setPower(0);
-            wheel3.setPower(0);
-            wheel4.setPower(0);
-        }
-        if (gamepad1.left_stick_x > -0.3 && gamepad1.left_stick_x < 0.3) {
-            wheel1.setPower(0);
-            wheel2.setPower(0);
-            wheel3.setPower(0);
-            wheel4.setPower(0);
-        }
-        if (gamepad1.left_stick_y < 0.3) {
-            wheel1.setPower(0.7);
-            wheel2.setPower(-0.7);
-            wheel3.setPower(0.7);
-            wheel4.setPower(-0.7);
-        }
-        if (gamepad1.left_stick_y > -0.3) {
-            wheel1.setPower(-0.7);
-            wheel2.setPower(0.7);
-            wheel3.setPower(-0.7);
-            wheel4.setPower(0.7);
-        }
-        if (gamepad1.left_stick_x < 0.3) {
-            wheel1.setPower(0.7);
-            wheel2.setPower(0.7);
-            wheel3.setPower(-0.7);
-            wheel4.setPower(-0.7);
-        }
-        if (gamepad1.left_stick_x > -0.3) {
-            wheel1.setPower(-0.7);
-            wheel2.setPower(-0.7);
-            wheel3.setPower(0.7);
-            wheel4.setPower(0.7);
-        }
+            //Neutral condition for horizontal
+        if (gamepad1.left_stick_x > -0.3 && gamepad1.left_stick_x < 0.3) { wheelMove (0, 0, 0, 0); }
```

Comment:
Why do you have this code twice?

Reply:
In the first push lines 116-121 had wheel1 set to all the powers but we changed it to wheel1, 2, 3, and 4, set to p1, p2, p3, and p4 respectively.

Reply:
No, I mean why are lines 41-42 the same as 44-45  😸

Reply:
Lines 41-42 set a vertical deadzone so we made it on the y axis while lines 44-45 set a horizontal deadzone so we made it on the x axis

Reply:
tesy=t